modules/ROOT/pages/batch-filters-and-batch-aggregator.adoc

```diff
@@ -190,26 +190,37 @@ Using random access, you can change a record's payload at any given index positi
 
 ==== Considerations for Defining a Block Size
 
-On a traditional on-line processing model, each request is usually mapped to a worker thread. Regardless of the processing type (either synchronous, asynchronous, one-way, request-response or even if the requests are temporarily buffered before being processed), servers usually end up in a 1:1 relationship between a request and a running thread. +
+In a traditional online processing model, each request is usually mapped to a worker thread. Regardless of the processing type (either synchronous, asynchronous, one-way, request-response or even if the requests are temporarily buffered before being processed), servers usually end up in a 1:1 relationship between a request and a running thread. +
 When it comes to a batch job, all records are first stored in a persistent queue before the Process phase begins, so that the traditional threading model wouldn't apply.
 
-To improve performance, the runtime queues and schedules batch records in blocks of 100 records. This lessens the amount of I/O requests and improves an operation's load. +
-The default threading profile of the runtime is 16 threads per job. Therefore, in a default configured batch job each of the 16 threads processes a block of 100 records. +
-Each thread iterates through that block processing each record, and then each block is queued back, and the process continues.
+To improve performance, the runtime queues and schedules batch records in blocks
```

Comment:
would make sense to replace "the runtime" with just "Mule" here?

Reply:
done